Gruntfile.js

```diff
@@ -153,6 +153,7 @@ module.exports = function (grunt) {
                 '/' + staticDir + '/geo.ext.min.js',
                 // '/' + rootStaticDir + '/libs.min.js', // libs included in jade template
                 '/' + staticDir + '/jquery.gridster.js',
+                '/' + staticDir + '/jquery-ui.min.js',
```

Comment:
@danlamanna, @aashish24 I noticed jquery-ui is already imported into geojs via bower, so I modified Gruntfile.js to copy them in from there.